### PR TITLE
This PR makes use of the new csi-lib-utils/rpc package.

### PR DESCRIPTION
**Pull request by MorrisLaw**
This PR updates livenessprobe to use, the new csi-lib-utils library, rpc.

### CODE REVIEW

**Comment by MorrisLaw** [comment]
@andrewsykim

File: cmd/livenessprobe/main.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```
Comment (jsafrane) [inline cmd/livenessprobe/main.go]: We don't update copyright  on updates (and no, I don't know why we have such strange rules).

Comment (MorrisLaw) [reply cmd/livenessprobe/main.go]: ACK, I'll update now

**Comment by jsafrane** [comment]
LGTM, please squash the commits.

**Pull request merged**